numpy/distutils/mingw32ccompiler.py

```diff
@@ -85,17 +85,28 @@ def __init__ (self,
         elif self.linker_dll == 'gcc':
             self.linker = 'g++'
 
-        # **changes: eric jones 4/11/01
-        # 1. Check for import library on Windows.  Build if it doesn't exist.
+        p = subprocess.Popen(['gcc', '--version'], shell=True,
+                             stdout=subprocess.PIPE)
```

Comment:
Doesn't look like that will work here without other modifications. In particular there is
```
if 'MinGW-W64' not in str(out_string):
```
And `-dump-version` might not deliver that. I'd rather not fool with this without having windows to check the work with.